docs/excel/custom-functions-runtime.md

```diff
@@ -64,9 +62,9 @@ ws.onerror = (error) => {
 }
 ```
 
-## Managing data with AsyncStorage
+## Storing and accessing data
 
-AsyncStorage is a persistent, unencrypted, key-value storage system that can be used to cache data. It can be used as an alternative to localStorage, which is not available to custom functions. AsyncStorage is accessible to custom functions as a global object and to all other parts of your add-in through `OfficeRuntime.AsyncStorage`. Each add-in has a 5MB storage partition by default.
+Within a custom function (or within any other part of an add-in), you can store and access data by using the `OfficeRuntime.AsyncStorage` object. `AsyncStorage` is a persistent, unencrypted, key-value storage system that provides an alternative to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), which cannot be used within custom functions. An add-in can store up to 5MB of data using `AsyncStorage`.
```

Comment:
Also - is this a good place to use which? Just curious if we should not use it at all or if this is an appropriate use case I should know about. (Totally ok if it just somehow slipped through the which hunt.)

Reply:
This instance of `which` is okay because it's the beginning of a clause and it cannot be replaced with `that` and still be proper grammar. In general, using `which` is permissible at the beginning of a clause like this (i.e., where the clause follows a comma and is describing something that came before it in the sentence -- in this case, it's describing localStorage).

Reply:
Awesome, thanks for the clarification!